docs/src/Groups/groups.md

```diff
@@ -22,6 +22,160 @@ Julia supports the following types of groups:
 
-## Permutations groups
 
-In Julia, permutations are displayed as products of disjoint cycles, as in GAP.
+If `x` is an element of the group `G` of type `T`, then the type of `x` is `GAPGroupElement{T}`.
+
+## Basics
+
+### Elements of groups
+
+Given a group `G`, it is always possible to have access to some particular elements.
+
+```@docs
+one(x::GAPGroup)
+rand(::GAPGroup)
+rand_pseudo(G::GAPGroup)
+gens(::GAPGroup)
+```
+
+If the group `G` has been created as the subgroup of another group generated by a list of elements `L`, then the generating set returned by the function `gens` corresponds to `L`.
+
+!!! note
+    The output of `gens(G)` is not, in general, the minimal list of generators for `G`.
+
+### Operations
+
+Oscar supports the following operations and functions on group elements.
+
+* `*` = multiplication between two elements in a group.
+* `inv(x)` = `x^-1` the inverse of `x`.
+* `x/y` = the element `x*y^-1`.
+* `x^n` = the ``n``-th power of `x`. If ``n = 0``, the identity of the group is returned; if ``n < 0 ``, the ``n``-th power of the inverse of `x` is returned.
+* `isone(x)`: returns whether `x` is the identity of the group.
+* `conj(x,y)` = `x^y` = the conjugate of `x` by `y`, i.e. the element `x^-1*y*x`.
+* `comm(x,y)` = the commutator of `x` and `y`, i.e. the element `x^-1*y^-1*x*y`.
+
+
+## Subgroups
+
+The subgroup of a group `G` generated by the elements `x,y,...` are defined by the following instruction:
+
+* `sub(G, [x,y,...])` ;
+* `sub(x,y,...)`.
+
+This function returns two objects: a group `H`, that is the subgroup of `G` generated by the elements `x,y,...`, and the embedding homomorphism of `H` into `G`. The object `H` has the same type of `G`, and it has no memory of the "parent" group `G`: it is an independent group.
+
+  **Examples:**
+```jldoctest
+julia> G = symmetric_group(4); H = sub(G,[cperm([1,2,3]),cperm([2,3,4])]);
+julia> H[1] == alternating_group(4)
+true
+```
+
+
+## Permutation groups
+
+Permutation groups can be defined as symmetric groups, alternating groups or their subgroups.
+
+```@docs
+symmetric_group
+alternating_group
+```
+
+Every permutation group `G` is identified by a degree, that corresponds ideally to the size of the set on which `G` is acting.
+
+!!! note
+    The degree of a group of permutations is not necessarily the highest moved point of the group `G`. For example, if `G` is defined as a   subgroup of `Sym(n)`, its degree is `n` even if every element of `G` fixes `n`.
+
+### Permutations
+
+Permutations in Oscar are displayed as products of disjoint cycles, as in GAP. An explicit permutation can be built using the functions `perm`, `gap_perm` and `cperm`.
+
+```@docs
+perm
+gap_perm
+cperm
+```
+
+  **Example:**
+```jldoctest
+julia> perm(symmetric_group(6),[2,4,6,1,3,5])
+(1,2,4)(3,6,5)
+
+julia> cperm([1,2,3],4:7)
+(1,2,3)(4,5,6,7)
+
+julia> cperm([1,2],[2,3])
+(1,3,2)
+```
+
+At the moment, the input vectors of the function `cperm` need not to be disjoint.
```

Comment:
This sounds like it should be inside the docstring of cperm, not here. Also, it is a bit unclear what the significance of this statement is? Should this behavior be surprising? If so, perhaps it should explained what happens in this case? Or perhaps it all works fine?

Reply:
It simply means that the function does not check whether the input cycles are disjoint (and, in that case, return error), as other programs do (e.g. GAP or Magma).